doc/nrf/config_and_build/companion_components.rst

```diff
@@ -0,0 +1,71 @@
+
+The following table lists the available companion components:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Component
+     - Purpose
+     - Supported platforms
+     - Location
+     - Integration instructions
+   * - :ref:`ipc_radio`
+     - Companion component that allows to use the radio peripheral from another core in a multicore device.
+     - | :ref:`nRF54H20 DK <ug_nrf54h20_gs>`
+       | :ref:`nRF5340 DK <ug_nrf5340>`
+       | :ref:`Thingy:53 <ug_thingy53>`
+     - :file:`applications/ipc_radio`
+     - :ref:`Configuration <ipc_radio>`
+   * - :ref:`MCUboot <mcuboot:mcuboot_wrapper>`
+     - Secure bootloader for 32-bit microcontrollers.
+     - :ref:`All supported boards <app_boards_names>`
+     - :file:`ncs/bootloader/mcuboot` (external module) and :file:`ncs/nrf/modules/mcuboot` (integration files)
+     - :ref:`mcuboot:mcuboot_ncs`
+   * - `Trusted Firmware-M (TF-M) <TF-M documentation_>`_
+     - Platform security architecture reference implementation aligning with PSA Certified guidelines, enabling chips, Real Time Operating Systems and devices to become PSA Certified.
+     - | :ref:`nRF91x1 Series devices <ug_nrf91>`
+       | :ref:`nRF54L15 PDK <ug_nrf54l>`
+       | :ref:`nRF54H20 DK <ug_nrf54h20_gs>`
+       | :ref:`nRF5340 DK <ug_nrf5340>`
+       | :ref:`Thingy:53 <ug_thingy53>`
+     - :file:`ncs/modules/tee/tf-m`
+     - :ref:`ug_tfm`
+   * - :ref:`bootloader`
+     - Bootloader tailored for the :ref:`two-stage bootloader <immutable_bootloader>`.
+     - See :ref:`bootloader_rot`
+     - :file:`samples/bootloader`
+     - See :ref:`ug_bootloader_adding_immutable_b0`
+   * - :ref:`SUIT flash companion <suit_flash_companion>`
+     - Companion image that allows the Secure Domain Firmware to access the external memory during the :ref:`Software Updates for Internet of Things (SUIT) <ug_nrf54h20_suit_dfu>` firmware upgrade.
+     - See :ref:`Sample requirements <suit_flash_companion>`
+     - :file:`samples/suit/flash_companion`
+     - See :ref:`Building and running section <suit_flash_companion>`
+   * - :ref:`SUIT flash recovery image <suit_recovery>`
+     - Companion image that allows recovering the device firmware if the original firmware is damaged during the :ref:`Software Updates for Internet of Things (SUIT) <ug_nrf54h20_suit_dfu>` firmware upgrade.
+     - See :ref:`Sample requirements <suit_recovery>`
+     - :file:`samples/suit/recovery`
+     - See :ref:`Building and running section <suit_recovery>`
+
+Adding companion components to your application
+***********************************************
+
+Integration of some companion components may require changing the configuration of your application.
+Refer to the component documentation for tested and supported integration instructions.
```

Comment:
```suggestion
Integration of some companion components may require changing the configuration of the companion component and your application.
Refer to the component documentation for tested and supported integration instructions.
```

Reply:
I moved lines around, but tried to convey this.